### ISSUE TITLE
Redundant lines in interactive DST explorer script

### ISSUE THREAD

**Issue by alexpearce** [lhcb/first-analysis-steps#141]
When students write their first DST explorer script, it looks like [the one here](https://github.com/lhcb/first-analysis-steps/blob/962d68424e5e8f7add0e22fe858832a3c656cd82/05-interactive-dst.md):

``` python
import sys

import GaudiPython as GP
from GaudiConf import IOHelper
from Configurables import (
  LHCbApp,
  ApplicationMgr,
  DataOnDemandSvc,
  DecodeRawEvent,
  CondDB,
  DaVinci
)


appConf = ApplicationMgr()

dv = DaVinci()
dv.DataType = '2012'

dre = DecodeRawEvent()
dre.DataOnDemand = True

lhcbApp = LHCbApp()
lhcbApp.Simulation = True
CondDB().Upgrade = False

# Pass file to open as first command line argument
inputFiles = [sys.argv[-1]]
IOHelper('ROOT').inputFiles(inputFiles)

appMgr = GP.AppMgr()
evt = appMgr.evtsvc()

appMgr.run(1)

evt.dump()
```

We can reduce that to this:

``` python
import sys

import GaudiPython as GP
from GaudiConf import IOHelper
from Configurables import DaVinci

dv = DaVinci()
dv.DataType = '2012'
dv.Simulation = True

# Pass file to open as first command line argument
inputFiles = [sys.argv[-1]]
IOHelper('ROOT').inputFiles(inputFiles)

appMgr = GP.AppMgr()
evt = appMgr.evtsvc()

appMgr.run(1)
evt.dump()
```

The diff:

``` diff
5,15c5
< from Configurables import (
<   LHCbApp,
<   ApplicationMgr,
<   DataOnDemandSvc,
<   DecodeRawEvent,
<   CondDB,
<   DaVinci
< )
< 
< 
< appConf = ApplicationMgr()

---
> from Configurables import DaVinci
19,25c9
< 
< dre = DecodeRawEvent()
< dre.DataOnDemand = True
< 
< lhcbApp = LHCbApp()
< lhcbApp.Simulation = True
< CondDB().Upgrade = False

---
> dv.Simulation = True
35d18
< 
```
## Rationale

We should keep everything as simple as possible. I had a hard time explaining the `DecodeRawEvent`, `ApplicationMgr`, and `CondDB` stuff, and now I'm _pretty_ certain they're superfluous anyway.

I've tested this with the [first interactive lesson](https://github.com/lhcb/first-analysis-steps/blob/962d68424e5e8f7add0e22fe858832a3c656cd82/05-interactive-dst.md) and everything's the same.

I am mildy in favour of specifying `DaVinci().CondDBtag` and `DaVinci().DDDBtag` in this file. It's really not necessary for looking around, but it is best practices to always define them when processing MC data .

**Comment by betatim** [comment]
:+1: for turning this into a PR. I am a fan of getting rid of fluff.

**Comment by betatim** [comment]
This got fixed as part of #146 apparently the auto closing of issues is not as smart as we think :)
